src/ui/Facet/FacetSettings.ts

```diff
@@ -186,6 +186,10 @@ export class FacetSettings extends FacetSort {
         this.unselectItem(this.getSortItem(criteria.name));
       }
     });
+
+    if (this.sortSection) {
```

Comment:
Why do we need to check if the `sortSection` is defined?

If there is a chance that it does not exist, could we focus on the first focusable element in the `settingsPopup` element instead? Otherwise, I imagine we would still have the same accessibility issue?

Reply:
If there are no enabled sorts, there will be no defined `sortSection`. I just checked again though and found that there are other focusable elements apart from those of `sortSection`, so I'll make it select whichever first element has `tabindex >= 0`.